libraries/gpu-gl-common/src/gpu/gl/GLTextureTransfer.cpp

```diff
@@ -218,7 +236,19 @@ void GLTextureTransferEngineDefault::updateMemoryPressure() {
 
     Backend::textureResourceIdealGPUMemSize.set(idealMemoryAllocation);
     size_t unallocated = idealMemoryAllocation - totalVariableMemoryAllocation;
-    float pressure = (float)totalVariableMemoryAllocation / (float)allowedMemoryAllocation;
+    float pressure = 0;
+
+    if ( useAvailableGlMemory ) {
+        float total_mem = GLBackend::getTotalMemory();
+        float avail_mem = GLBackend::getAvailableMemory() - AUTO_RESERVE_TEXTURE_MEMORY;
+        if ( avail_mem < 0 ) {
```

Comment:
```suggestion
        if (avail_mem < 0) {
```

Reply:
also getAvailableMemory returns a size_t so this can never be negative

Reply:
The issue is the  `- AUTO_RESERVE_TEXTURE_MEMORY;` bit on line 243. My thought is that we should ensure some memory is always left unallocated, because something else on the system might need some.